tests/ops/models/test_property.py

```diff
@@ -29,24 +42,62 @@ def minimal_experience(
         experience_config.save(db=db)
         return {"id": experience_config.id, "name": experience_config.name}
 
-    def test_create_property(self, db, minimal_experience):
+    def test_create_property(self, db, minimal_experience, privacy_center_config):
         prop = Property.create(
             db=db,
             data=PropertySchema(
                 name="New Property",
                 type=PropertyType.website,
                 experiences=[minimal_experience],
+                privacy_center_config=privacy_center_config,
+                stylesheet=":root:root { --chakra-colors-gray-50: #fff9ea; }",
+                paths=["test"],
             ).dict(),
         )
         assert prop.name == "New Property"
         assert prop.type == PropertyType.website
         assert prop.id.startswith("FDS")
+        assert prop.privacy_center_config == privacy_center_config
+        assert prop.stylesheet == ":root:root { --chakra-colors-gray-50: #fff9ea; }"
+        assert prop.paths == ["test"]
         assert len(prop.experiences) == 1
 
         experience = prop.experiences[0]
         assert experience.id == minimal_experience["id"]
         assert experience.name == minimal_experience["name"]
 
+        prop.delete(db)
+
+    def test_create_property_duplicate_paths(
+        self, db, minimal_experience, privacy_center_config
+    ):
+        first_prop = Property.create(
+            db=db,
+            data=PropertySchema(
+                name="First Property",
+                type=PropertyType.website,
+                experiences=[minimal_experience],
+                privacy_center_config=privacy_center_config,
+                stylesheet=":root:root { --chakra-colors-gray-50: #fff9ea; }",
+                paths=["test"],
+            ).dict(),
+        )
+
+        with pytest.raises(IntegrityError):
+            Property.create(
+                db=db,
+                data=PropertySchema(
+                    name="Second Property",
+                    type=PropertyType.website,
+                    experiences=[minimal_experience],
+                    privacy_center_config=privacy_center_config,
+                    stylesheet=":root:root { --chakra-colors-gray-50: #fff9ea; }",
+                    paths=["test"],
+                ).dict(),
+            )
+
+        first_prop.delete(db)
```

Comment:
haven't verified this test specifically, but from what i can see with testing on the API, i think the `Second Property` actually gets created here _before_ the `IntegrityError` is raised, so i think this test could/should be adjusted to ensure that the `Second Property` isn't actually created!

Reply:
Updated test